src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java

```diff
@@ -115,6 +122,20 @@ public class JavadocParagraphCheck extends AbstractJavadocCheck {
      */
     public static final String MSG_MISPLACED_TAG = "javadoc.paragraph.misplaced.tag";
 
+    /**
+     * A key is pointing to the warning message text in "messages.properties"
+     * file.
+     */
+    public static final String MSG_EXTRA_TAG = "javadoc.paragraph.preceded.block.tag";
```

Comment:
Please put `.preceded.block.tag` in name
MSG_preceded_block_tag

Reply:
renamed